pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -160,7 +157,7 @@ func (r *Reconciler) reconcile(ctx context.Context, ks *servingv1alpha1.KnativeS
 // Transform the resources
 func (r *Reconciler) transform(instance *servingv1alpha1.KnativeServing) (*mf.Manifest, error) {
 	r.Logger.Debug("Transforming manifest")
-	transforms, err := platform.Transformers(r.KubeClientSet, instance, r.Logger)
+	transforms, err := r.platform.Transformers(r.KubeClientSet, instance, r.Logger)
```

Comment:
Can r.platform be nil?

Reply:
Yes `r.platform` can be nil. In this case, only default transformers are returned.